src/mixins/nr-affiliation-mixin.ts

```diff
@@ -89,11 +89,11 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
           return
         }
 
-        this.setAffiliationErrorModalValue(NrAffiliationErrors.UNABLE_TO_START_REGISTRATION)
+        this.setAffiliationErrorModalValue(NrAffiliationErrors.ASSOCIATED_OTHER_ACCOUNT)
```

Comment:
If we get to this point then we know the NR is already affiliated, but when we fetched the affiliations for the current account, we didn't find a matching name request affiliation or temporary business affiliation -- so the most likely scenario is that the NR is affiliated to a different account.